buildSrc/src/main/kotlin/kotlin-multiplatform-conventions.gradle.kts

```diff
@@ -63,6 +63,18 @@ kotlin {
             api("org.jetbrains.kotlinx:atomicfu-wasm-js:${version("atomicfu")}")
         }
     }
+    @OptIn(org.jetbrains.kotlin.gradle.targets.js.dsl.ExperimentalWasmDsl::class)
+    wasmWasi {
+        nodejs()
+        compilations["main"]?.dependencies {
+            api("org.jetbrains.kotlinx:atomicfu-wasm-wasi:${version("atomicfu")}")
+        }
+        compilations.configureEach {
```

Comment:
Unfortunately we can't do this in `configure-compilation-conventions.gradle.kts` because there is no way to distingish `WasmWasi` target from `WasmJs` or `JS` targets there.